Gordon360/Services/RecIM/SeriesService.cs

```diff
@@ -623,21 +680,47 @@ public async Task<EliminationRound> ScheduleElimRoundAsync(IEnumerable<SeriesTea
             var series = _context.Series.Find(involvedTeams.First().SeriesID);
 
             var teams = involvedTeams.Reverse();
-
+            var matches = new List<MatchViewModel>();
+            var byeTeams = new List<int>();
+            //bye logic
             while (!(((numTeams + numByes) != 0) && (((numTeams + numByes) & ((numTeams + numByes) - 1)) == 0))) //while not power of 2
```

Comment:
while what is not the power of 2?

If you want to keep this comment, it could probably be a little bit more descriptive.

Reply:
non-blocking